.github/workflows/integrationTest.yml

```diff
@@ -879,4 +891,80 @@ jobs:
 #          max_attempts: 3
 #          timeout_minutes: 8
 #          retry_wait_seconds: 5
-#          command: cd terraform/ec2/linux && terraform destroy --auto-approve
\ No newline at end of file
+#          command: cd terraform/ec2/linux && terraform destroy --auto-approve
+
+  EC2AcceptanceIntegrationTest:
+    needs: [ECSFargateIntegrationTest, ECSEC2LaunchDaemonIntegrationTest, EC2LinuxIntegrationTest, GenerateTestMatrix]
```

Comment:
If we run this action after this list of actions, we're entirely dependent on the stability of the integration tests. Also, `GenerateTestMatrix` is redundant because it's a transitive dependency